src/System.Memory/ref/System.Memory.cs

```diff
@@ -178,7 +178,8 @@ public unsafe struct MemoryHandle : IDisposable
     {
         public MemoryHandle(IRetainable owner, void* pinnedPointer = null,  System.Runtime.InteropServices.GCHandle handle = default(System.Runtime.InteropServices.GCHandle))  { throw null; }
         public void* PinnedPointer { get { throw null; } }
-        public void Dispose()  { throw null; }
+        public void AddOffset(int offset) { throw null; }
```

Comment:
Has this been API reviewed?

Reply:
> If this is purely to help with the implementation of Memory, maybe it should just be internal?

I agree. I think this could be an internal API that is only needed by Memory.

@KrzysztofCwalina, do you see a scenario where we would need to expose the AddOffset API publically?

Reply:
(And if for some reason its functionality must be public, I'd prefer if we just made PinnedPointer settable.)

Reply:
Yes, the scenario is I use OwnedMemory and MemoryHandle directly, without using Memory, e.g. I implement my own Memory-like factory for spans. If we don't like AddOffset, we should add OwnedMemory.Pin overload that takes offset to the added to the pointer inside the handle.

Reply:
> Having said that, we need to fix this bug today, and so possibly an internal API is a better change short term so that we don't have to rush the API review.

The CI system is still down it seems. I am not sure if we can get this in today.